controllers/event.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const app = express.Router();
```

Comment:
it's more common to call this variable `router` since you already have an `app` variable in your app.js